litmus-portal/frontend/src/components/Header/styles.ts

```diff
@@ -1,18 +1,184 @@
 import { makeStyles, Theme } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
+  // Styles for Header
   appBar: {
     width: '100%',
     height: '100%',
     backgroundColor: theme.palette.background.default,
   },
-  breadCrumbsGrid: {
-    marginTop: theme.spacing(2),
-    paddingLeft: theme.spacing(4),
+
+  headerFlex: {
+    height: '5%',
```

Comment:
I guess we can't use '%' if it's not '100%' according to the guidelines.

Reply:
I don't think that's the case. Actually to make it flexible with the screen resolutions it's better to use %. theme.spacing wouldn't be very gradual.